rollup.config.js

```diff
@@ -94,6 +94,10 @@ export function makeBaseBundleConfig(options) {
       },
     },
     include: ['*.ts+(|x)', '**/*.ts+(|x)', '../**/*.ts+(|x)'],
+    // the typescript plugin doesn't handle concurrency very well, so clean the cache between builds and don't complain
+    // when old cache files can't be found (see https://github.com/ezolenko/rollup-plugin-typescript2/issues/15)
+    clean: true,
+    verbosity: 0,
```

Comment:
The clean option seems good to me. I am slightly worried about the verbosity option being set to 0 because it might hide away relevant errors in the future.

If we don't set the verbosity level to 0, will the build fail in the event cache files are missing (couldn't repro myself)? If it doesn't fail, I would actually be in favor of leaving it as-is and just have a bunch of warnings in the logs. If it actually fails - I'm fine with having it at 0. Bit annoying but what other options are there :p

Reply:
Now (of course), it's not doing it for me, either. Either it's flaky (though it was consistently mad before), or in spite of the cache cleaning there was still some remnant somewhere of something which I've since cleaned out (I've done a full wipe of the repo at least once in the meantime). ¯\\_(ツ)_/¯ I'm fine to stick it back at 1 for the moment and see if it starts spamming us again. I'll add a note to the file.